.github/workflows/connector_integration_test_single_dagger.yml

```diff
@@ -6,10 +6,14 @@ on:
       test-connectors-options:
         description: "Options to pass to the 'airbyte-ci connectors test' command"
         default: "--modified"
-  # pull_request:
-  #   paths:
-  #     - "airbyte-integrations/connectors/**"
-  #     - ".github/workflows/connector_integration_test_single_dagger.yml"
+  pull_request:
+    paths:
+      #- "airbyte-integrations/connectors/**"
+      #- ".github/workflows/connector_integration_test_single_dagger.yml"
```

Comment:
Due to current infra problems it's not safe to enable the automated test pipeline right now - concurrent jobs are blocking each other.

Reply:
@alafanechere if im reading this right then this workflow will run on "all" opened/ready for review PRs

Do we not want to keep these path filters so it runs on "all connector related" prs instead?

Reply:
This pipeline will eventually run on all PRs to make the required check pass...